vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -238,6 +232,10 @@ def parse_container_definition(container_def, pod_id)
       new_result[:container_port_configs] = Array(ports).collect do |port_entry|
         parse_container_port_config(port_entry, pod_id, container_def.name)
       end
+      containers = pod.spec.containers
+      containers.collect do |containers|
+        new_result[:container] = parse_container(containers, pod_id)
+      end
```

Comment:
Same, if you modify the code above according to my comments then you don't need this change here.